python/llm/example/CPU/LangChain/README.md

```diff
@@ -1,45 +1,70 @@
-## Langchain Examples
+# Langchain examples
```

Comment:
Please update readme for install prerequisites. Reference: https://github.com/intel-analytics/ipex-llm/tree/main/python/llm/example/GPU/LlamaIndex